js/app.js

```diff
@@ -1,41 +1,49 @@
-// --- API KEY: AIzaSyCKNNUXyaTDWJp14VhcwraU-mY7o4A27HY --- //
+// Global Variables
+var tubeKey = 'AIzaSyCKNNUXyaTDWJp14VhcwraU-mY7o4A27HY';
```

Comment:
Be very careful with API keys and Git / GitHub.  Now that this is posted here you should consider this key to be completely public.  That may be ok here, but be careful to avoid doing this with secret keys or other credentials.